scripts/multi_reg_and_predict_new_data.py

```diff
@@ -0,0 +1,163 @@
+import glob
+import csv
+import sys
+import re
+import numpy as np
+from sklearn import linear_model
+
+# This script trains on in.*.dat in the training directory passed on the
+# command-line and then predicts on in.*.dat in the working directory.
+
+def setupRegressors(files, bool_cols, y_col, rank_col, excluded, X, Y, ranks, regressor_list, n_obs):
+    first_row = True
```

Comment:
Variable `first_row` is defined before for loop. This means that this variable will be True  just for first read `file_name` in `files`. Is it desired behaviour? Or maybe it should be after `for file_name in files:` but before `with open(file_name, ...`, then it would be True for each read file.